apidoc/View.yml

```diff
@@ -277,8 +277,30 @@ methods:
   - name: removeAllCircles
     summary: Remove all circles from the map.
 
-
+  - name: setClusterAnnotation
```

Comment:
We should add an example of using clusters in the `examples:` section of the YML.

Reply:
Good idea. Added example.